integrations/GA4/browser.js

```diff
@@ -15,13 +15,17 @@ import { NAME } from "./constants";
 
 export default class GA4 {
   constructor(config, analytics) {
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
     this.measurementId = config.measurementId;
     this.analytics = analytics;
     this.sendUserId = config.sendUserId || false;
     this.blockPageView = config.blockPageViewEvent || false;
     this.extendPageViewParams = config.extendPageViewParams || false;
     this.extendGroupPayload = config.extendGroupPayload || false;
     this.debugMode = config.debugMode || false;
+    this.isHybridModeEnabled = !config.useNativeSDKToSend;
```

Comment:
What happens if `useNativeSDKToSend` is `null` or `undefined`?

Reply:
updated